### DISCUSSION TITLE
Which issue tracker should we add support for next?

### DISCUSSION THREAD

**Discussion by Ygnys** [GitLiveApp/GitLive#139] [Polls]
GitLive currently integrates with Jira, GitHub Issues, Bitbucket Issues, Azure DevOps Work Items and GitLab Issues.

Here you can vote for the issue tracker we should add support for next, or add/upvote a comment below if your preferred issue tracker is missing from the poll.

**Comment by codehampster** [comment]
Can you please support Azure DevOps
